x-pack/plugins/ml/public/application/routing/routes/explorer.tsx

```diff
@@ -260,27 +268,31 @@ const ExplorerUrlStateManager: FC<ExplorerUrlStateManagerProps> = ({ jobsWithTim
           </EuiFlexItem>
         </EuiFlexGroup>
       </MlPageHeader>
-      <AnomalyExplorerContext.Provider value={anomalyExplorerContext}>
-        {jobsWithTimeRange.length === 0 ? (
-          <AnomalyDetectionEmptyState />
-        ) : (
-          <Explorer
-            {...{
-              explorerState,
-              overallSwimlaneData,
-              showCharts,
-              severity: tableSeverity.val,
-              stoppedPartitions,
-              invalidTimeRangeError,
-              selectedJobsRunning,
-              timeBuckets,
-              timefilter,
-              selectedCells,
-              swimLaneSeverity,
-            }}
-          />
-        )}
-      </AnomalyExplorerContext.Provider>
+      <StyledComponentsThemeProvider>
+        <CasesContext owner={[]} permissions={casesPermissions!}>
```

Comment:
`cases` is an optional plugin, should we be asserting not undefined here?

Reply:
no need, this is how `CasesContext` is initialized. 
```
const CasesContext = cases?.ui.getCasesContext() ?? React.Fragment;
```